packages/carousel/src/react/use-swipe.js

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useRef, useState } from 'react'
+
+const noop = () => {}
+
+const defaultOpts = {
+  distanceThreshold: 20, // 20px
```

Comment:
rename with px suffix

Reply:
i opted not to change this because these variables are part of the hook api and it didn't seem right to include the unit suffix